net.sourceforge.qtpfsgui.LuminanceHDR.yaml

```diff
@@ -0,0 +1,99 @@
+app-id: net.sourceforge.qtpfsgui.LuminanceHDR
+runtime: org.kde.Platform
+runtime-version: "5.11"
+sdk: org.kde.Sdk
+command: luminance-hdr
+rename-desktop-file: luminance-hdr.desktop
+rename-appdata-file: luminance-hdr.appdata.xml
+rename-icon: luminance-hdr
+finish-args:
+  - "--socket=fallback-x11"
+  - "--filesystem=home"
+modules:
+  - name: exiv2
+    cleanup: [/bin]
+    buildsystem: cmake
+    sources:
+      - type: git
+        url: "https://github.com/Exiv2/exiv2/"
+        commit: e7ffd83af29187190da7f98dcbca8a4d70e19582
+
+  - name: fftw3
+    buildsystem: autotools
```

Comment:
Not needed, it's the default buildsystem.

Reply:
IMHO, it's better to state it explicitly, just in case default will change as well as others could understand what is going on easier (without the need to remember what are the defaults).

Reply:
I wouldn't expect the default to ever change, but its not a blocker or anything if prefered.